pkg/signal/signal_unix.go

```diff
@@ -0,0 +1,99 @@
+// +build aix darwin dragonfly freebsd netbsd openbsd solaris zos
```

Comment:
Are we sure this is safe? I see no guarantees anywhere that signal name->number mappings are the same across OSes?

Reply:
@mheon so this code is just a copy of the _unsupported variant which had all these linux mappings, so it’s what is in effect today before this change on all those platforms. The only difference is just that SIGWINCH is aliased to the actual syscall value as opposed to the 0xff placeholder. I didn’t investigate if and how those values are actually used in a non Linux remote usage (other than ensuring compilation can succeed), my assumption is they might end up being referenced as part of passed state during remote calls with Linux containers (would explain Linux values).